libs/feed/social_security_feed.py

```diff
@@ -0,0 +1,79 @@
+from abc import ABC
+from bs4 import BeautifulSoup
```

Comment:
Add to requirements.txt all depends libs

Reply:
Done @liranzxc